splinter/driver/flaskclient.py

```diff
@@ -57,10 +57,11 @@ class FlaskClient(LxmlDriver):
 
     driver_name = "flask"
 
-    def __init__(self, app, user_agent=None, wait_time=2):
+    def __init__(self, app, user_agent=None, wait_time=2, custom_headers=None):
```

Comment:
This was mimicked after `DjangoClient` which uses `custom_headers`:

https://github.com/cobrateam/splinter/blob/6993e85e48088c3f6546e2e4be83d52dce863ea5/splinter/driver/djangoclient.py#L65